packages/app/src/sections/dashboard/Stake/EscrowTab.tsx

```diff
@@ -1,30 +1,172 @@
+import { formatPercent, formatTruncatedDuration, truncateNumbers } from '@kwenta/sdk/utils'
+import { useMemo } from 'react'
+import { useTranslation } from 'react-i18next'
 import styled from 'styled-components'
 
+import { FlexDivCol, FlexDivRow, FlexDivRowCentered } from 'components/layout/flex'
+import { Body, Heading } from 'components/Text'
+import { NO_VALUE } from 'constants/placeholder'
+import { useAppSelector } from 'state/hooks'
+import {
+	selectAPY,
+	selectAPYV2,
+	selectStakedEscrowedKwentaBalance,
+	selectStakedEscrowedKwentaBalanceV2,
+	selectStakedResetTime,
+	selectTotalVestable,
+	selectTotalVestableV2,
+} from 'state/staking/selectors'
 import media from 'styles/media'
 
+import { StakingCard } from './card'
 import EscrowTable from './EscrowTable'
 import EscrowInputCard from './InputCards/EscrowInputCard'
 
 const EscrowTab = () => {
+	const { t } = useTranslation()
+	const apy = useAppSelector(selectAPY)
+	const apyV2 = useAppSelector(selectAPYV2)
+	const stakedEscrowedKwentaBalance = useAppSelector(selectStakedEscrowedKwentaBalance)
+	const stakedEscrowedKwentaBalanceV2 = useAppSelector(selectStakedEscrowedKwentaBalanceV2)
+	const totalVestable = useAppSelector(selectTotalVestable)
+	const totalVestableV2 = useAppSelector(selectTotalVestableV2)
+	const stakedResetTime = useAppSelector(selectStakedResetTime)
+
+	const timeLeft = useMemo(
+		() =>
+			stakedResetTime > new Date().getTime() / 1000
+				? formatTruncatedDuration(stakedResetTime - new Date().getTime() / 1000)
+				: NO_VALUE,
+		[stakedResetTime]
+	)
+
+	const DEFAULT_CARDS = [
```

Comment:
be good to change to useMemo and change name to regular casing

Reply:
Thanks. Updated.